topics/RDMA.md

```diff
@@ -0,0 +1,170 @@
+---
+title: "RDMA support"
+linkTitle: "RDMA support"
+description: Valkey Over RDMA support
+---
+
+Valkey supports the Remote Direct Memory Access (RDMA) connection type via a
+Valkey module that can be dynamically loaded on demand.
+
+## Getting Started
+
+RDMA enables direct data exchange between networked computers' main memory,
+bypassing processors and operating systems.
+
+As a result, RDMA offers better performance compared to TCP. Test results indicate that
+Valkey Over RDMA achieves approximately 2 times higher QPS and lower latency.
+
+Please note that Valkey Over RDMA is currently supported only on Linux.
+
+## Running manually
+
+To run a Valkey server with RDMA mode:
+
+    ./src/valkey-server --protected-mode no \
+         --loadmodule src/valkey-rdma.so bind=192.168.122.100 port=6379
+
+Bind address/port of RDMA can be modified at runtime using the following command:
+
+    192.168.122.100:6379> CONFIG SET rdma-port 6380
+
+Valkey can run both RDMA and TCP concurrently on the same port:
+
+    ./src/valkey-server --protected-mode no \
+         --loadmodule src/valkey-rdma.so bind=192.168.122.100 port=6379 \
+         --port 6379
+
+Note that the network interface (192.168.122.100 of this example) should support
+RDMA. To test a server supports RDMA or not:
+
+    ~# rdma dev show (a new version iproute2 package)
+Or:
+
+    ~# ibv_devices (ibverbs-utils package of Debian/Ubuntu)
+
+
+## Protocol
+
+The protocol defines the QP type RC (like TCP), communication commands, and payload exchange mechanism.
+This dependency is based solely on the RDMA (aka Infiniband) specification
```

Comment:
What dependency are you referring to here?

Reply:
Like TCP/IP, Valkey defines `RESP protocol` over TCP/IP, but not dependent on Network interface vendor.

I want to say: Valkey Over RDMA depends on RDMA only, but not any low level transport or vendor.
Please help me to make this easy to understand...